### ISSUE TITLE
All In for Maintainers DEI Resource Hub 

### ISSUE THREAD

**Issue by soyetubo** [AllInOpenSource/All-In#28]
# All In for Maintainers DEI Resource Hub 

tl;dr
In  issue #24 @Demetris shared details on the program planning pertaining to the launch of the All In for Maintainers program. This issue provides details specifically pertaining to the creation of a central hub for DEI resources.

**The primary objective:** create a central repository for vetted DEI resources for maintainers to easily search and access.
All In for Maintainers will launch the All In for Maintainers DEI Resource Hub in Q3 (anticipated ship: Feb 1, 2023).

The All In for Maintainers DEI Resource Hub will include two key ships spanning across a 6-month timeline: a repository of resources from readily available content; and the creation of new DEI-focused content through the All In Creators initiative.
Key metrics will include: the number of site visits and resources sourced and created.

---

## All In for Maintainers DEI Resource Hub
**_What is the DEI Resource Hub?_**
The All In for Maintainers DEI Resource Hub is a highly curated and searchable list of current and relevant DEI best practices, training material, research and trends for open source maintainers.

Very similar in model to [https://resources.github.com/] (https://resources.github.com/) and [www.opensource.com](http://www.opensource.com/), the DEI Resource Hub seeks to provide a digital archive of currently available and newly created open source specific DEI resources from various organizations, thought leaders and researchers. 

To achieve these objectives, we have identified two net-new programs that will launch in the first half of 2023. 

1. A repository of resources from already created content: 
2. All In Creators original content

### _Why is this necessary?_
During the early stages of [All In](https://github.com/AllInOpenSource/All-In/issues/allinopensource.org), we learned that maintainers and community leaders really set the tone for inclusion within their respective communities. After connecting with nearly 300 maintainers and community leaders as a part of the [Maintainers Listening Tour](https://allinopensource.org/community/), a common challenge shared by participants was:

> "The number of DEI resources we have is overwhelming. We wish there was a website or repo where we could go to get all of the resources in one place."

The All In for Maintainers DEI Resource Hub is a solution to begin building solutions to solve the pain points maintainers have identified as blocks in their journey of learning how to create more diverse and inclusive open source communities.

### _Who is this resource designed for?_
Though designed in response to the feedback we received from maintainers, this resource hub is for anyone interested in learning more about current trends, research, best practices and more, related to advancing DEI in open source. 

### _What are the key topic areas for the Resource Hub?_

- Community Hospitality
- Contributions
- Meetings 
- Marketing and Communications
- Code of Conduct
- Intervention and De-escalation Strategies
- Inclusive Language
- Cultural considerations
- Inclusive Leadership
- Unconscious Bias for Maintainers
- Others?
- Topics that align with the metrics used for the All In Badge initiative

### _Where will these resources be hosted?_
A curated list of resources vetted and approved by the All In editorial team, alongside the future All In originals content, will live on the All In Community page and All In Repo. 

### _When will these initiatives ship?_
The All In for Maintainers DEI Resource Hub, a repository of resources from readily available content, will ship February 1 2023.

All In Creators - a new program to create original content relating to DEI in open source and address the gaps in research - will launch its open RFP in June 2023 as a part of Maintainers Month. Keep an eye out for a new issue with more details on the All In Creators program to come soon.

### _How will this project tie into the broader All In for Maintainers program?_

The goal with this Resource Hub is to help maintainers educate themselves around opportunities for improvement that were identified through the Badging Program. This set of resources should support them in their DEI journey as they move up through the different Badging levels. For example, perhaps during the badging process, they might realize that they need more information on how to best attend to measuring contributor demographics in their project. The Resource Hub would be a good place for them to start.

Additionally, the All In Badging program can help identify potential gaps in resources and tools for maintainers, which will help us understand what topic areas and types of content are still needed in the Resource Hub. 

Tracking metrics around the Resource Hub will also provide insight into the areas where maintainers struggle the most, which could identify potential opportunities for tooling through the Grant piece or opportunities for Cross-Project Collaboration through All in for Maintainers.

### _How can I get involved?_
We invite you to join us in open-sourcing diversity, equity, and inclusion. You can do so by sharing recommendations for publicly available resources that are relevant; offer practical, actionable advice; and are from trusted sources. Please note that everything shared on the site will only be done so after receiving permission from the original authors to republish, and we are committed to attributing accordingly. If you’re interested in suggesting a resource to be included, please fill out the [form](https://docs.google.com/forms/d/1bE-yjBCLJGQr0WGVqHEjZO2C1cTyVVtcND4zouMMYlM/edit). 

### _How will you measure success?_

- We will measure success by tracking the following key metrics:
- Tools created 
- New content created through the All In Creators program (open RFP)
- Number of resources
- Number of views on resources provided
- Community polling on the usefulness and relevance of the resources

---

We truly appreciate your interest in contributing to the All In for Maintainers DEI Resource Hub. As always, we welcome feedback and suggestions on how to make this initiative more impactful!

### _Please note the following:_

- The DEI Resource Hub is not meant to be an exhaustive list of all published DEI in tech resources
- We appreciate suggestions from the community for new resources, but the All In editorial team may decide to include only those that align with the All In mission.

**Comment by jancborchardt** [comment]
Note there is a grassroots resource collection and community forum similar to what this sounds like at https://opensourcediversity.org :) Maybe there is some collaboration opportunity?

cc @jonatoni @kristiprogri @sleepypioneer @alicetragedy @erioldoesdesign @gretad
